ui/src/main/webapp/src/app/reports/technologies/tech-report.service.ts

```diff
@@ -119,6 +125,50 @@ export class TechReportService extends GraphService
         return result;
     }
 
+    getHibernateEntityModel(execID: number): Observable<HibernateEntityModel[]> {
+        const entitiesObservable = this.getTypeAsArray<HibernateEntityModel>(HibernateEntityModel.discriminator, execID, {
+            /*
+             * TODO: THIS IS PROBLEM
+             *
+             * Endpoint expects to get label of edge, but frontend has it ONLY in @GraphAdjacency annotation
+             *  which is not accessible.
+             *
+             *  It is not straightforward that 'PersistenceEntity-jpaEntityClass' will resolve to 'javaClass'
+             */
+            out: this.getProperiesString('PersistenceEntity-jpaEntityClass', 'decompiledSource')
```

Comment:
I think we did add some handling of edges already. IIRC the Java model has to have a specific annotation not to skip the edge. Jess will remember more, or it's used somewhere.

Reply:
Yeah, I'd have to dig through the code comments and usecases to remember it though.

Reply:
Comment is pointing out to something else. It works nicely - endpoint returns model with resolved properties specified in `out` option. 

But the thing is `PersistenceEntityModel` in Java looks like this:
```java
    String TYPE = "PersistenceEntity";
    String JPA_ENTITY_CLASS = TYPE + "-jpaEntityClass";
    /**
     * Contains the entity class
     */
    @Adjacency(label = JPA_ENTITY_CLASS, direction = Direction.OUT)
    JavaClassModel getJavaClass();
``` 

which results into this TypeScript method: 
```typescript
    @GraphAdjacency('PersistenceEntity-jpaEntityClass', 'OUT', false, 'ADJACENCY')
    get javaClass(): Observable<JavaClassModel> { return null; };
```
As you see, what used to be static variable in java (`JPA_ENTITY_CLASS`) is now hardcoded as string into TypeScript annotation. 

Unfortunately, to resolve `javaClass` from graph endpoint, you have to specify `PersistenceEntity-jpaEntityClass` (which is name of the edge) and not `javaClass`.